src/vs/workbench/api/common/extHostExtensionService.ts

```diff
@@ -795,7 +795,7 @@ export abstract class AbstractExtHostExtensionService extends Disposable impleme
 	private async _activateAndGetResolver(remoteAuthority: string): Promise<{ authorityPrefix: string; resolver: vscode.RemoteAuthorityResolver | undefined }> {
 		const authorityPlusIndex = remoteAuthority.indexOf('+');
 		if (authorityPlusIndex === -1) {
-			throw new Error(`Not an authority that can be resolved!`);
+			throw RemoteAuthorityResolverError.InvalidAuthority(`Not an authority that can be resolved!`);
```

Comment:
We should actually construct this manually; the class in extHostTypes is exposed to extensions and they won't throw this type of error.

```suggestion
			throw new RemoteAuthorityResolverError(`Not an authority that can be resolved!`, RemoteAuthorityResolverErrorCode.InvalidAuthority);
```

that's the reason it was done down below